crates/ibc/src/clients/ics07_tendermint/client_state.rs

```diff
@@ -792,19 +792,102 @@ fn check_header_trusted_next_validator_set(
     }
 }
 
+#[cfg(test)]
+pub mod test_utils {
```

Comment:
is it being used anywhere else? if not, I would suggest to merge this block under `mod test { ... }`.

Reply:
[e88452c](https://github.com/cosmos/ibc-rs/pull/954/commits/e88452cff23461b00ecf08ea60f4d11a67e89449)